### PR TITLE
FrontEnd: Basics of Internet - Hosting and Name Servers 

### LINKED ISSUE #2357: FrontEnd: Basics of Internet - Hosting and Name Servers
### Description

Give Documentation of Hosting and Name Servers

### Domain

Frontend Dev HTML/CSS/JS

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by Yashesvinee**
<hr>

## Description 📜

Added documentation on Hosting and Name Servers
Fixes #2357 

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Frontend Dev HTML/CSS/JS

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by Yashesvinee** [comment]
@2024-SANDHYA please review this PR

File: Web_Development/FrontEnd/Hosting_name_servers/documentation.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+
+# Hosting and Name Servers
+----
```
Comment (praveenscience) [inline Web_Development/FrontEnd/Hosting_name_servers/documentation.md:3]: No need of so many HRs.

File: Web_Development/FrontEnd/Hosting_name_servers/documentation.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+
```
Comment (praveenscience) [inline Web_Development/FrontEnd/Hosting_name_servers/documentation.md:1]: Remove Blank Lines.

File: Web_Development/FrontEnd/Hosting_name_servers/documentation.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+
+# Hosting and Name Servers
+----
+## What is it?
+   Hosting, in simple words, is a service through which resources are provided to an individual or organization for the maintenance and storage of websites or other related services. Hosting is something that you will require when your are creating a website. It  will provide you the storage space reqiured for storing your site's data like videos, images, applications etc.<br><img src="img/img1.jpg" width="400" align="center"><br>
+   A Name Server is a server component of the DNS that returns the IP Address of a given Domain name. It handles queries regarding the location of the domain name’s various services.
+
+
```
Comment (praveenscience) [inline Web_Development/FrontEnd/Hosting_name_servers/documentation.md:8]: Remove Blank Lines.

File: Web_Development/FrontEnd/Hosting_name_servers/documentation.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+
+# Hosting and Name Servers
+----
+## What is it?
+   Hosting, in simple words, is a service through which resources are provided to an individual or organization for the maintenance and storage of websites or other related services. Hosting is something that you will require when your are creating a website. It  will provide you the storage space reqiured for storing your site's data like videos, images, applications etc.<br><img src="img/img1.jpg" width="400" align="center"><br>
```
Comment (praveenscience) [inline Web_Development/FrontEnd/Hosting_name_servers/documentation.md:5]: Add space after heading.

Comment (praveenscience) [reply Web_Development/FrontEnd/Hosting_name_servers/documentation.md:5]: Do not have more spaces in front of text.

File: Web_Development/FrontEnd/Hosting_name_servers/documentation.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+
+# Hosting and Name Servers
+----
+## What is it?
+   Hosting, in simple words, is a service through which resources are provided to an individual or organization for the maintenance and storage of websites or other related services. Hosting is something that you will require when your are creating a website. It  will provide you the storage space reqiured for storing your site's data like videos, images, applications etc.<br><img src="img/img1.jpg" width="400" align="center"><br>
+   A Name Server is a server component of the DNS that returns the IP Address of a given Domain name. It handles queries regarding the location of the domain name’s various services.
```
Comment (praveenscience) [inline Web_Development/FrontEnd/Hosting_name_servers/documentation.md:6]: Do not have more spaces in front of text.

File: Web_Development/FrontEnd/Hosting_name_servers/documentation.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
+
+# Hosting and Name Servers
+----
+## What is it?
+   Hosting, in simple words, is a service through which resources are provided to an individual or organization for the maintenance and storage of websites or other related services. Hosting is something that you will require when your are creating a website. It  will provide you the storage space reqiured for storing your site's data like videos, images, applications etc.<br><img src="img/img1.jpg" width="400" align="center"><br>
+   A Name Server is a server component of the DNS that returns the IP Address of a given Domain name. It handles queries regarding the location of the domain name’s various services.
+
+ 
+ ----
```
Comment (praveenscience) [inline Web_Development/FrontEnd/Hosting_name_servers/documentation.md:9]: No need of so many HRs.

File: Web_Development/FrontEnd/Hosting_name_servers/documentation.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
+
+# Hosting and Name Servers
+----
+## What is it?
+   Hosting, in simple words, is a service through which resources are provided to an individual or organization for the maintenance and storage of websites or other related services. Hosting is something that you will require when your are creating a website. It  will provide you the storage space reqiured for storing your site's data like videos, images, applications etc.<br><img src="img/img1.jpg" width="400" align="center"><br>
+   A Name Server is a server component of the DNS that returns the IP Address of a given Domain name. It handles queries regarding the location of the domain name’s various services.
+
+ 
+ ----
+ ## Web Hosting
```
Comment (praveenscience) [inline Web_Development/FrontEnd/Hosting_name_servers/documentation.md:10]: Add space after heading.

File: Web_Development/FrontEnd/Hosting_name_servers/documentation.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+   A Name Server is a server component of the DNS that returns the IP Address of a given Domain name. It handles queries regarding the location of the domain name’s various services.
+
+ 
+ ----
+ ## Web Hosting
+ Web Hosting is provided by companies that rent out their technologies and services to host your website. Once the Host provider hosts your website, users can access it by typing in your web address (domain name) in their browser. Their computer connects to the server your website is hosted on and the server in turn displays the stored files to your web visitor.<br>There are many Web hosts you can choose from and hence, research is required to find the ideal one for your website. You must check the Host provider's performance, security, scalability, etc.<br> So how do you go about this?
+ - Decide the type of website you want to build, it could be static or dynamic. Static websites show the same content to all users while dynamic websites show contents that involve the user's activity on that website. Both these types may require different hosting features
```
Comment (praveenscience) [inline Web_Development/FrontEnd/Hosting_name_servers/documentation.md:12]: Add space before list.

**Review by praveenscience — CHANGES_REQUESTED** [review]
Please repeat all the changes over the whole content. Use a program like Typora to do this.

**Comment by Yashesvinee** [comment]
@praveenscience sir I made the necessary changes
Is it okay now?

**Comment by 2024-SANDHYA** [comment]
> 
> 
> @2024-SANDHYA please review this PR

There already exists a PR for this, so you either change this to audio or video as a doc for the same topic will not be accepted, as informed in the discord itself.

**Comment by Yashesvinee** [comment]
@2024-SANDHYA there is a PR for domain name basics and DNS
Could you please tell me PR number for Hosting and name servers
I am sorry for asking but I just want to confirm

**Comment by Yashesvinee** [comment]
@2024-SANDHYA Please let me know if this is a valid PR 
I took up this issue after checking if there were similar ones and there weren't any

**Comment by 2024-SANDHYA** [comment]
> 
> 
> @2024-SANDHYA Please let me know if this is a valid PR I took up this issue after checking if there were similar ones and there weren't any

Check it for yourself !

**Pull request closed without merging**